cypress/e2e/settings/users/patron-block/patron-block-maximum-lost-items-removed-after-lost-item-renewed.cy.js

```diff
@@ -0,0 +1,316 @@
+    UserLoans.updateTimerForAgedToLost('reset');
+    cy.getAdminToken();
+    cy.get('@items').each((item) => {
+      CheckInActions.checkinItemViaApi({
+        itemBarcode: item.barcode,
+        servicePointId: testData.userServicePoint.id,
+        checkInDate: new Date().toISOString(),
+      });
+    });
+    NewFeeFine.getUserFeesFines(userData.userId).then((userFeesFines) => {
+      const feesFinesData = userFeesFines.accounts;
+      feesFinesData.forEach(({ id }) => {
+        cy.deleteFeesFinesApi(id);
+      });
+    });
+    cy.get('@items').each((item, index) => {
+      cy.deleteItem(item.itemId);
+      cy.deleteHoldingRecordViaApi(itemsData.itemsWithSeparateInstance[index].holdingId);
+      InventoryInstance.deleteInstanceViaApi(itemsData.itemsWithSeparateInstance[index].instanceId);
+    });
+    PaymentMethods.deleteViaApi(testData.paymentMethodId);
+    UsersOwners.deleteViaApi(testData.ownerId);
+    cy.deleteLoanPolicy(loanPolicyBody.id);
+    LostItemFeePolicy.deleteViaApi(lostItemFeePolicyBody.id);
+    CirculationRules.deleteRuleViaApi(originalCirculationRules);
+    cy.deleteLoanType(testData.loanTypeId);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.userServicePoint.id]);
+    ServicePoints.deleteViaApi(testData.userServicePoint.id);
+    Users.deleteViaApi(userData.userId);
+    PatronGroups.deleteViaApi(patronGroup.id);
+    Conditions.resetConditionViaApi('72b67965-5b73-4840-bc0b-be8f3f6e047e', 'Maximum number of lost items');
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id
+    );
+  });
+  it(
+    'C350653 Verify automated patron block "Maximum number of lost items" removed after lost item renewed (vega)',
+    { tags: [TestTypes.criticalPath, devTeams.vega] },
+    () => {
+      cy.visit(SettingsMenu.conditionsPath);
+      Conditions.waitLoading();
+      Conditions.select('Maximum number of lost items');
+      Conditions.setConditionState(blockMessage);
+      cy.visit(SettingsMenu.limitsPath);
+      Limits.selectGroup(patronGroup.name);
+      Limits.setLimit('Maximum number of lost items', '4');
+      // needed for the "Lost Item Fee Policy" so items can get "aged to lost" status
+      cy.wait(230000);
+
+      cy.visit(TopMenu.usersPath);
+      UsersSearchPane.waitLoading();
+      UsersSearchPane.searchByKeywords(userData.barcode);
+      UsersCard.waitLoading();
+      Users.checkIsPatronBlocked(blockMessage, 'Borrowing, Renewals, Requests');
+
+      const itemForRenew = itemsData.itemsWithSeparateInstance[Math.floor(Math.random() * 5)];
+      UsersCard.openLoans();
+      UsersCard.showOpenedLoans();
+      UserLoans.openLoan(itemForRenew.barcode);
+      UserLoans.renewItem(itemForRenew.barcode, true);
+      Renewals.renewBlockedPatron(`AutotestText${getRandomPostfix()}`);
```

Comment:
Maybe move parametr of funcition to const?)

Reply:
done